edx_proctoring/static/proctoring/js/views/proctored_exam_add_bulk_allowance_view.js

```diff
@@ -201,7 +201,8 @@ edx = edx || {};
                     },
                     error: function(unused, response) {
                         var data = $.parseJSON(response.responseText);
-                        $errorResponse.html(gettext(data.detail));
+                        var errorField = data.field;
+                        self.showError(self, data.field, data.detail);
```

Comment:
I think this line is readable as-is, you can probably remove the `errorField` variable.